packages/shared/src/hooks/companion/useBackgroundRequest.ts

```diff
@@ -14,12 +14,13 @@ export const useBackgroundRequest = (
   const client = useQueryClient();
 
   useRawBackgroundRequest(({ key, ...args }) => {
-    if (!enabled || !isQueryKeySame(key, queryKey)) {
+    const validKey = Array.isArray(key) ? key : [key];
+    if (!enabled || !isQueryKeySame([validKey], queryKey)) {
```

Comment:
Isn't this now double array since `key` is always wrapped in array above as `validKey`?

Reply:
Good point, yes, modified! 🙏